db/e2e-test-db.sql

```diff
@@ -0,0 +1,801 @@
+
+
+--
+-- Name: django_content_type id; Type: DEFAULT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public.django_content_type ALTER COLUMN id SET DEFAULT nextval('public.django_content_type_id_seq'::regclass);
+
+
+--
+-- Name: django_migrations id; Type: DEFAULT; Schema: public; Owner: postgres
+--
+
+ALTER TABLE ONLY public.django_migrations ALTER COLUMN id SET DEFAULT nextval('public.django_migrations_id_seq'::regclass);
+
+
+--
+-- Data for Name: auth_group; Type: TABLE DATA; Schema: public; Owner: postgres
+--
+
+COPY public.auth_group (id, name) FROM stdin;
+\.
+
+
+--
+-- Data for Name: auth_group_permissions; Type: TABLE DATA; Schema: public; Owner: postgres
+--
+
+COPY public.auth_group_permissions (id, group_id, permission_id) FROM stdin;
+\.
+
+
+--
+-- Data for Name: auth_permission; Type: TABLE DATA; Schema: public; Owner: postgres
+--
+
+COPY public.auth_permission (id, name, content_type_id, codename) FROM stdin;
+1	Can add permission	1	add_permission
+2	Can change permission	1	change_permission
+3	Can delete permission	1	delete_permission
+4	Can view permission	1	view_permission
+5	Can add group	2	add_group
+6	Can change group	2	change_group
+7	Can delete group	2	delete_group
+8	Can view group	2	view_group
+9	Can add content type	3	add_contenttype
+10	Can change content type	3	change_contenttype
+11	Can delete content type	3	delete_contenttype
+12	Can view content type	3	view_contenttype
+13	Can add user	4	add_account
+14	Can change user	4	change_account
+15	Can delete user	4	delete_account
+16	Can view user	4	view_account
+\.
+
+
+--
+-- Data for Name: authentication_account; Type: TABLE DATA; Schema: public; Owner: postgres
+--
+
+COPY public.authentication_account (id, password, is_superuser, email, username, cmtee_id, contact, first_name, last_name, role, tagline, delete_ind, created_at, updated_at, last_login, is_staff, is_active, date_joined, login_code_counter, register_token, personal_key, status, code_generated_counter, secret_key) FROM stdin;
+\.
+
+
+--
+-- Data for Name: authentication_account_groups; Type: TABLE DATA; Schema: public; Owner: postgres
+--
+
+COPY public.authentication_account_groups (id, account_id, group_id) FROM stdin;
+\.
+
+
+--
+-- Data for Name: authentication_account_user_permissions; Type: TABLE DATA; Schema: public; Owner: postgres
+--
+
+COPY public.authentication_account_user_permissions (id, account_id, permission_id) FROM stdin;
+\.
+
+
+--
+-- Data for Name: django_content_type; Type: TABLE DATA; Schema: public; Owner: postgres
+--
+
+COPY public.django_content_type (id, app_label, model) FROM stdin;
+1	auth	permission
+2	auth	group
+3	contenttypes	contenttype
+4	authentication	account
+\.
+
+
+--
+-- Data for Name: django_migrations; Type: TABLE DATA; Schema: public; Owner: postgres
+--
+
+COPY public.django_migrations (id, app, name, applied) FROM stdin;
```

Comment:
i would be in favor of having a clean migrations table and then running migrate on the django instance if that works.